deepmd/descriptor/se_t.py

```diff
@@ -477,7 +477,7 @@ def _filter(self,
                 env_i = tf.reshape(inputs_i, [-1, nei_type_i, 4])
                 # with natom x nei_type_i x 3
                 env_i = tf.slice(env_i, [0, 0, 1], [-1, -1, -1])
-                start_index_j = 0
+                start_index_j = start_index_i - self.sel_a[type_i]
```

Comment:
How about moving the assignment of `start_index_j` before 472 the update of `start_index_i`, so we can write 
`start_index_j = start_index_i`

Reply:
sure